src/Bicep.Core/Emit/EmitterSettings.cs

```diff
@@ -51,6 +52,11 @@ public EmitterSettings(SemanticModel model)
         /// </summary>
         public bool UseExperimentalTemplateLanguageVersion { get; }
 
+        /// <summary>
+        /// Enables bicep local deploy
+        /// </summary>
+        public bool LocalDeployEnabled { get; }
```

Comment:
Since this is just a convenience property, and you don't want the value to differ from what is on the model, could we replace it with the following?

```csharp
public bool LocalDeployEnabled => model.Features.LocalDeployEnabled;
```

Reply:
I moved to use `Context.SemanticModel.Features.LocalDeployEnabled` instead.